src/drive/web/modules/views/Folder/FolderViewBody.jsx

```diff
@@ -36,7 +36,10 @@ const FolderViewBody = ({
     setSortOrder({ sortAttribute: attribute, sortOrder: order })
   )
 
-  const isInError = queryResults.some(query => query.fetchStatus === 'error')
+  const isInError = queryResults.some(query => {
+    console.log('query', query)
```

Comment:
Careful, a log is here.

Reply:
look at the associated commit message ^^"

Reply:
I don't know why the test is green on my machine and not on travis.